src/manifest.json

```diff
@@ -8,9 +8,16 @@
         "128": "img/sugarcube.png"
     },
 
+    "browser_action": {
+        "default_icon": "img/sugarcube.png"
+    },
+
     "devtools_page": "html/devtools.html",
 
     "permissions": [
+        "webRequest",
+        "*://*/*/sidecar.js*",
+        "*://*/*/sidecar.min.js*",
```

Comment:
I need these to check when someone is requesting these urls in any domain and any scheme